src/Reference_SQL_One.php

```diff
@@ -117,6 +125,20 @@ function ($m) {
         return $ex;
     }
 
+    public function ref($defaults = [])
+    {
+        $m = parent::ref($defaults);
+
+        // If model is not loaded, then we are probably doing deep traversal
```

Comment:
only "probably" but that's not always the case.

How can I get all orders of active clients?
```
$c = new Client();
$c->addCondition('active', true);
$o = $c->ref('Order');
```
what will this return?